ACM_General/home/static/home/css/officers.css

```diff
@@ -1,148 +1,135 @@
+.nav-officers a span {
+  color: #4d4d4d !important;
 }
 
-.main-section .content-wrapper:nth-of-type(2) {
-    padding-top: 20px;
+.nav-officers:hover a span {
+  color: #4d4d4d !important;
 }
 
-.main-section .content-wrapper h1 {
-    margin-top: 0;
+.nav-officers:after {
+  transform: scaleX(.8);
 }
 
-.main-section .content-wrapper h2 {
-    margin: 0;
+@media (max-width: 450px) {
+  .btn {
+    font-size: 0.7em;
+  }
 }
 
-.main-section .content-wrapper h2 a {
-    display: inline;
-    margin:0;
-}
-
-@media(max-width: 450px) {
-    .btn {
-        font-size: 0.7em;
-    }
+@media (max-width: 900px) {
+  .cards__item {
+    width: 100% !important;
+  }
+  
+  .card {
+    width: 100%;
+  }
 }
 
-@media(max-width: 900px) {
-    .cards__item {
-        width: 100% !important;
-    }
-    .card {
-        width: 100%;
-    }
-}
 .link_zone {
-    display: flex;
+  display: flex;
 }
+
 .btn {
-    background-color: white;
-    border: 1px solid lightgray;
-    //border-radius: 1rem;
-    padding: 0.5rem 0;
-    font-weight: bold;
-    color: #77A22F;
-    text-decoration: none;
-    text-align: center;
-    /*font-size: .9em;*/
-    /*padding-left: 22px;*/
+  padding: 8px 0;
+  /*font-size: .9em;*/
+  /*padding-left: 22px;*/
 }
 
 .btn--block {
-    display: block;
-    width: 31%;
-    margin:0 1.1666666px;
+  display: block;
+  width: 31%;
+  margin: 0 1.1666666px;
 }
 
 .cards {
-    display: flex;
-    flex-wrap: wrap;
-    list-style: none;
-    margin: 0;
-    padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0;
+  padding: 0;
 }
 
 .cards__item {
-    display: flex;
-    padding: 1rem;
-    width: 50%;
+  display: flex;
+  padding: 1rem;
+  width: 50%;
 }
 
 .card {
-    background-color: white;
-    border-radius: 0.25rem;
-    /*box-shadow: 0 20px 40px -14px rgba(0,0,0,0.25);*/
-    display: flex;
-    flex-direction: row;
-    //overflow: hidden;
-    max-height: 270px;
-    width: 100%;
+  background-color: #f1f1f1;
+  /*border-radius: 0.25rem;*/
+  /*box-shadow: 0 20px 40px -14px rgba(0,0,0,0.25);*/
+  display: flex;
+  flex-direction: row;
+/ / overflow: hidden;
+  max-height: 270px;
+  width: 100%;
 }
 
 .card__content {
-    display: flex;
-    flex: 1 1 auto;
-    flex-direction: column;
-    padding: 1rem;
-    width: 60%;
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  padding: 1rem;
+  width: 60%;
 }
 
 .card__image {
-    background-position: center center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    border-top-left-radius: 0.25rem;
-    border-top-right-radius: 0.25rem;
-    filter: contrast(70%);
-    //filter: saturate(180%);
-    overflow: hidden;
-    position: relative;
-    transition: filter 0.5s cubic-bezier(.43,.41,.22,.91);
-    max-width: 155px;
-    min-width: 155px;
-    height: 100%;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  /*border-top-left-radius: 0.25rem;*/
```

Comment:
If these comments are necessary in the CSS, could you please put a little blurb about them explaining why they are commented out.